src/github.com/getlantern/detour/whitelist.go

```diff
@@ -19,7 +19,9 @@ var (
 func AddToWl(addr string, permanent bool) {
 	muWhitelist.Lock()
 	defer muWhitelist.Unlock()
-	whitelist[addr] = wlEntry{permanent}
+	if add != "" {
```

Comment:
A small safety check. I realised some getParentDomain() parameters could return "" in some specific cases.